test/unit/misc/SummarisedMultiSelector.spec.js

```diff
@@ -19,17 +19,94 @@ const vm = new Constructor({
 }).$mount()
 
 describe('summarised multi selector', () => {
-    it('should match the snapshot', () => {
-        expect(vm.$el).toMatchSnapshot()
+    describe('none selected', () => {
+        it('should match the snapshot', () => {
+            expect(vm.$el).toMatchSnapshot()
+        })
+
+        describe('read only', () => {
+            beforeEach(() => {
+                vm.readOnly = true
+            })
+
+            it('should match the snapshot', () => {
+                expect(vm.$el).toMatchSnapshot()
+            })
+        })
     })
 
-    describe('read only', () => {
-        beforeEach(() => {
-            vm.readOnly = true
+    describe('some selected', () => {
+        describe('build summary', () => {
+            describe('from array', () => {
+                it('should handle single value', () => {
+                    vm.value = [1]
+                    vm.$nextTick(() => {
+                        expect(vm.selectedItems).toBe('PPC')
+                        expect(vm.displayText).toBe('PPC')
+                    })
+                })
+
+                it('should handle mutliple values', () => {
+                    vm.value = [1, 2]
+                    vm.$nextTick(() => {
+                        expect(vm.selectedItems).toBe('PPC, Paid Social')
+                        expect(vm.displayText).toBe('2 Selected')
+                    })
+                })
+
+                it('should match the snapshot', () => {
+                    vm.value = [1, 2]
+                    expect(vm.$el).toMatchSnapshot()
+                })
+            })
+
+            describe('from string', () => {
+                it('should handle single value', () => {
+                    vm.value = '1'
+                    vm.$nextTick(() => {
+                        expect(vm.selectedItems).toBe('PPC')
+                        expect(vm.displayText).toBe('PPC')
+                    })
+                })
+
+                it('should handle mutliple values', () => {
```

Comment:
grammar error here

Reply:
Thanks for that, it is fixed now.
Are we happy for this to be merged in?